.jenkins/common_job_properties.groovy

```diff
@@ -220,4 +220,38 @@ class common_job_properties {
       mailer(notifyAddress, false, true)
     }
   }
+
+  // Configures the argument list for performance tests, adding the standard
+  // performance test job arguments.
+  private static def genPerformanceArgs(def argMap) {
+    def standard_args = [
+      project: 'apache-beam-testing',
+      dpb_log_level: 'INFO',
+      dpb_maven_binary: '/home/jenkins/tools/maven/latest/bin/mvn',
+      bigquery_table: 'beam_performance.pkb_results',
+      official: 'true'
+    ]
+    def joined_args = standard_args.plus(argMap)
```

Comment:
What wins in case of key collision? This can also be a comment about argMap, rather than the behavior of `plus`